README.md

```diff
@@ -4,52 +4,56 @@ At Recurrent, we work with data we collect from people’s electric vehicles
 (EVs). Currently, we collect between one and three data points a day (usually)
 from each vehicle. The data we collect includes the following information:
 
- - `charge_reading`: The vehicle’s State of Charge (SoC), a decimal number
-   representing the current charge level of the battery in terms of percentage.
-(For example, 0.33 means the battery is 33% charged.)
- - `range_estimate`: The estimated range that the vehicle can drive before the
-   battery is depleted, expressed in miles as a decimal number.
- - `odometer`: The vehicle’s current odometer reading, expressed in miles as an
-   integer.
- - `plugged_in`: Whether or not the vehicle is plugged in at the time of the
-   reading, expressed as a boolean.
- - `charging`: Whether or not the vehicle is actively charging at the time of
-   the reading, expressed as a boolean.
- - `created_at`: Timestamp of the reading, expressed in a string of the format
-   "yyyy-mm-dd hh:mm:ss", the timezone is UTC for all readings.
- - `vehicle_id`: A unique identifier tied to the vehicle, expressed as a
-   string.
-
-The comma-separated-values(csv) file in this repo contains a collection of data
-points over the course of a few weeks for 5 different vehicles.
+- `charge_reading`: The vehicle’s State of Charge (SoC), a decimal number
+  representing the current charge level of the battery in terms of percentage.
+  (For example, 0.33 means the battery is 33% charged.)
+- `range_estimate`: The estimated range that the vehicle can drive before the
+  battery is depleted, expressed in miles as a decimal number.
+- `odometer`: The vehicle’s current odometer reading, expressed in miles as an
+  integer.
+- `plugged_in`: Whether or not the vehicle is plugged in at the time of the
+  reading, expressed as a boolean.
+- `charging`: Whether or not the vehicle is actively charging at the time of
+  the reading, expressed as a boolean.
+- `created_at`: Timestamp of the reading, expressed in a string of the format
+  "yyyy-mm-dd hh:mm:ss", the timezone is UTC for all readings.
+- `vehicle_id`: A unique identifier tied to the vehicle, expressed as a
+  string.
```

Comment:
My editor indented some of this stuff, but there were no content changes up here.